README.md

```diff
@@ -138,15 +144,18 @@ To use the tool on a MySQL database called mydb, run
 mysqldump mydb | $GOPATH/bin/harbourbridge -driver=mysqldump
 ```
 
+To use the tool on a DynamoDB database, run
+
+```sh
+$GOPATH/bin/harbourbridge -driver=dynamodb
+```
 More details on running harbourbridge can be found in [Example usage](#example-usage) section.
 
 This command will use the cloud project specified by the `GCLOUD_PROJECT`
 environment variable, automatically determine the Cloud Spanner instance
 associated with this project, convert the PostgreSQL/MySQL schema for `mydb` to a
```

Comment:
remove Postgres/Mysql so the line just reads convert the schema to a spanner schema